packages/utils/src/array.ts

```diff
@@ -66,20 +66,23 @@ export function addOrRemoveItem (array, value, included) {
   }
 }
 
-export function makeArray (arrayLikeObject) {
+export function makeArray<T=any> (arrayLikeObject:ArrayLike<T>):T[] {
   return Array.from(arrayLikeObject)
 }
 
-export function range (min, max) {
-  min = typeof min === 'function' ? min() : min
-  max = typeof max === 'function' ? max() : max
-  var result = []
-  for (var i = min; i <= max; i++) { result.push(i) }
+//TODO May be MaybeSubscribable<number> -> I actually don't want the dependency
```

Comment:
?? comment

Reply:
In the KO-type-def it is 
```typescript
export function range(min: MaybeSubscribable<number>, max: MaybeSubscribable<number>): number[];
``` 
I have change this to `() =>number | number` for auto unboxing.